### PR TITLE
Use CSS Modules in Graph System

### PR DESCRIPTION
**Pull request by carolhmj**
This is part of the work to make the graph system more reusable.

### CODE REVIEW

File: packages/dev/buildTools/src/webpackTools.ts [hunk 1]
```diff
@@ -119,7 +120,9 @@ export const getRules = (
                         loader: "css-loader",
                         options: {
                             sourceMap: true,
-                            modules: true,
+                            modules: {
+                                localIdentName: options.mode === "development" ? "[path][name]__[local]" : "[hash:base64]",
```
Comment (sebavan) [inline packages/dev/buildTools/src/webpackTools.ts:124]: This considers prod being the default (if nothing is passed in mode) whereas the mode a bit later in the files default to dev :-)

**Review by sebavan — CHANGES_REQUESTED** [review]
Looks good to me aside of the tiny issue :-) but I ll let @RaananW do the final review

File: packages/dev/buildTools/src/webpackTools.ts (hunk 1 above)
Comment (carolhmj) [reply packages/dev/buildTools/src/webpackTools.ts]: Goood catch Seb!

**Review by RaananW — COMMENTED** [review]
Thank you so much for that :-)

I just want to be sure that it doesn't influence the existing tools before merging. Does it change anything in the node editor/gui editor? As it was previously globally applied, it is possible that those classes that were removed are used in one of the tools.

Otherwise - approved, great work

**Review by sebavan — APPROVED** [review]
Looks all good to me, @RaananW can we merge it ?

**Comment by RaananW** [comment]
I'll wait with the merge until azure is stable, but yes, we can merge

**Pull request merged**